vms/propertyfx/mint_operation.go

```diff
@@ -36,8 +36,8 @@ func (op *MintOperation) Outs() []verify.State {
 }
 
 func (op *MintOperation) Verify() error {
-	switch {
-	case op == nil:
+	switch op {
```

Comment:
These switch case changes are pretty dubious... The reason they were done the way they were before was to be consistent with the other `Verify` functions which have many verification conditions... If we don't care about consistency here, then I think basically all the cases where the linter suggested changes should be converted into `if else` checks.

Reply:
Swapped out w/ if-else checks where it made sense